enhancements/sig-architecture/15-resourcebasedscheduling/README.md

```diff
@@ -0,0 +1,191 @@
+# Resource Based Scheduling
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management-io/website/)
+
+## Summary
+
+The proposed work would enhance the Placement APIs to support scheduling based on the resource usage/capacity of managed clusters.
+
+## Motivation
+
+When scheduling placement, select managed clusters based on cluster resource usage/capacity to prevents placement of workloads on clusters that have insufficient resources to actually run the workloads.
+
+Link to the feature request: https://github.com/open-cluster-management-io/community/issues/52
+
+### User Stories
+
+#### Story 1: User is able to specify resources in placement spec to favor clusters with larger allocatable to capacity ratio of the resources.
+  - User wants to deploy an application on a cluster whose CPU usage is less than others;
+  - User wants to deploy an application on 2 clusters whose allocatable to capacity ratio of both CPU and memory is larger than others;
+
+#### Story 2: User is able to specify resources in placement spec to favor clusters with larger amount of resources allocatable.
+  - User wants to deploy an application on a cluster with largest memory allocatable;
+
+#### Story 3: User is able to create a placement to select clusters based on resource usage, and then keep placement decisions updated according to the changes on cluster resource usage.
+  - A CPU-intensive application would like to be deployed on the cluster with least CPU utilization rate.
+
+#### Story 4: User is able to create a placement to select clusters based on resource usage, and then ignore any resource usage change afterwards to keep the placement decisions pinning.
```

Comment:
> Hi @elgnay ,
> 
> I have some comments. Most of them are rewording. Please consider them.
> Hope it can be helpful to make this doc more readable.

@dongwangdw Thanks for the comments. I'll update the proposal accordingly.